tests/models/test_taskinstance.py

```diff
@@ -1249,16 +1255,16 @@ def test_depends_on_past(self, dag_maker):
             pytest.param(
                 "all_done_setup_success",
                 2,
-                _UpstreamTIStates(6, 0, 1, 0, 0, 7, 1, 0),
+                [SUCCESS, SUCCESS, SUCCESS, SUCCESS, SUCCESS, SUCCESS, FAILED],
                 True,
                 (True, None),  # is_teardown=True, expect_state=None
                 True,
                 id="is teardown one setup failed one setup success",
             ),
             pytest.param(
-                "all_done_setup_success",
+                "all_done",
                 2,
-                _UpstreamTIStates(6, 0, 1, 0, 0, 7, 1, 0),
+                [SUCCESS, SUCCESS, SUCCESS, SUCCESS, SUCCESS, SUCCESS, FAILED],
```

Comment:
This pytest fails because it seems the dependency context is returned empty after evaluation. I am not sure why. I assume something is not "correctly mocked".